admin_guide/scheduling/custom_scheduler.adoc

```diff
@@ -0,0 +1,109 @@
+[[admin-guide-sched-custom]]
+= Custom Schedulers
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+
+{product-title} supports the use of custom (your own or third-party) pod schedulers alongside the default scheduler that comes installed. While the default scheduler should be sufficient for most situations, there might be times when you want to use a different scheduler. You can run multiple schedulers alongside the default scheduler and configure which scheduler to use for each pods. 
```

Comment:
I feel like, out of all of the above, only the last sentence is relevant. I'd suggest removing all except that.